Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -89,7 +89,7 @@ public RepositoryItemKey MappedBusinessFlow
         /// <summary>
         /// Been used to identify if UnMappedUIElements were lazy loaded already or not
         /// </summary>
-        public bool UnMappedUIElementsLazyLoad { get { return mUnMappedElements.LazyLoad; } }
+        public bool UnMappedUIElementsLazyLoad { get { return (mUnMappedElements != null) ? mUnMappedElements.LazyLoad : false; } }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5022389)